libc/cmake/modules/CheckCompilerFeatures.cmake

```diff
@@ -3,7 +3,7 @@
 # ------------------------------------------------------------------------------
 
 # Initialize ALL_COMPILER_FEATURES as empty list.
-set(ALL_COMPILER_FEATURES "float128")
+set(ALL_COMPILER_FEATURES "float128" "fixed_point")
```

Comment:
the comment implies this is supposed to be an empty list. Is the comment wrong or should these not be added at this point?

Reply:
Thanks for noticing!  I removed the comment above.